sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -4,316 +4,689 @@
-        #type(AttestationType, **Any) -> AttestationResponse[str]:
-        """ Retrieves the attestation policy for a specified attestation type.
+    def get_policy(self, attestation_type, **kwargs):
+        # type: (Union[str, AttestationType], **Any) -> Tuple[str, AttestationToken]
+        """Retrieves the attestation policy for a specified attestation type.
 
-        :param attestation_type: :class:`azure.security.attestation.AttestationType` for 
+        :param attestation_type: :class:`azure.security.attestation.AttestationType` for
             which to retrieve the policy.
-        
-        :type attestation_type: azure.security.attestation.AttestationType 
 
-        :return: Attestation service response encapsulating a string attestation policy.
-
-        :rtype: azure.security.attestation.AttestationResponse[str]
+        :type attestation_type: Union[str, ~azure.security.attestation.AttestationType]
+        :keyword bool validate_token: if True, validate the token, otherwise return the token unvalidated.
+        :keyword validation_callback: Function callback to allow clients to perform custom validation of the token.
+            if the token is invalid, the `validation_callback` function should throw
+            an exception.
+        :paramtype validation_callback: Callable[[~azure.security.attestation.AttestationToken, ~azure.security.attestation.AttestationSigner], None]
+        :keyword bool validate_signature: if True, validate the signature of the token being validated.
+        :keyword bool validate_expiration: If True, validate the expiration time of the token being validated.
+        :keyword str issuer: Expected issuer, used if validate_issuer is true.
+        :keyword float validation_slack: Slack time for validation - tolerance applied
+            to help account for clock drift between the issuer and the current machine.
+        :keyword bool validate_issuer: If True, validate that the issuer of the token matches the expected issuer.
+        :keyword bool validate_not_before_time: If true, validate the "Not Before" time in the token.
+
+        :return: A tuple containing the attestation policy and the token returned
+            by the service..
+
+        :rtype: Tuple[str, ~azure.security.attestation.AttestationToken]
 
         :raises azure.security.attestation.AttestationTokenValidationException: Raised when an attestation token is invalid.
 
+        .. note::
+            The Azure Attestation Policy language is defined `here <https://docs.microsoft.com/azure/attestation/author-sign-policy>`_
+
+        .. admonition:: Example: Retrieving the current policy on an attestation instance.
+
+
+            .. literalinclude:: ../samples/sample_get_set_policy.py
+                :start-after: [BEGIN get_policy]
+                :end-before: [END get_policy]
+                :language: python
+                :dedent: 8
+                :caption: Getting the current policy document.
+
+
+
+
```

Comment:
A lot of this whitespace can be omitted:
```suggestion
        .. admonition:: Example: Retrieving the current policy on an attestation instance.

            .. literalinclude:: ../samples/sample_get_set_policy.py
                :start-after: [BEGIN get_policy]
                :end-before: [END get_policy]
                :language: python
                :dedent: 8
                :caption: Getting the current policy document.

```

Reply:
I thought that sphinx required that literalinclude values be indented from admonitions, so I'm not sure how to reduce this.